subsys/bluetooth/common/Kconfig.template.log_config_bt

```diff
@@ -1,6 +1,19 @@
 # Copyright (c) 2022 Nordic Semicoductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+# Usage:
+# This template provides backwards compatibility for legacy kconfig symbols.
+# Do not add new uses of it.
+#
+# The following arguments are mandatory:
+# 	- module:
+#		Name of the new log level kconfig.
+#		Example: "BT_HCI_CORE"
+# 	- legacy-debug-sym:
+#		An existing "legacy" kconfig bool. If that bool is selected,
+#		the new kconfig is forced to max verbosity.
+#		Example: "BT_DEBUG_HCI_CORE"
```

Comment:
Can these be added as actual requirements? The template seems to build fine without supplying e.g. `legacy-debug-sym`, and simply added these as `mandatory` in a comment doesn't really seem to make much sense if it isn't validated in any way.

Reply:
I am pretty sure that Kconfig will complain if they are not defined, because they are being used in the template

Reply:
The reason why this was added, I presume, is specifically because I used this template _without_ setting a `legacy-debug-sym` in https://github.com/zephyrproject-rtos/zephyr/pull/52599, and I don't see any warnings there.

Reply:
There was no problem in your PR because you `legacy-debug-sym` was set previously in your Kconfig file.
Look here: https://github.com/zephyrproject-rtos/zephyr/blob/e457613f7ee4a5047bf6a84c6d518f23a6333f90/subsys/bluetooth/audio/Kconfig.baps#L272

Reply:
Ah, so the problem was not visible since I declared multiple log modules in same file?